.travis.yml

```diff
@@ -83,6 +83,6 @@ jobs:
       after_success: skip
 
 after_success:
-  - julia -e 'if Sys.islinux() && string(VERSION)[1:3] == "1.4"
```

Comment:
Abel, do you have an idea for avoiding this update at each new release of Julia ?

Reply:
Use just 1 ?!

Reply:
`1` is only used by Travis. `VERSION` is a variable defined by Julia and it is 1.0.5, 1.4.2, 1.5.0, etc...